recipes/hidefix/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge/label/rust_dev,conda-forge
```

Comment:
Any reason in particular to get the rust_dev version?

Reply:
I think hidefix requires nightly

Reply:
Looks like rust_dev is currently at `1.72.0.dev20230606`, and main is at `1.71.1`. If you're after rust 1.72, then should be ok.

Reply:
Ok yeah, just tested compiling locally, and got this error when using the stable `rust`:
```python-traceback
  error: could not compile `hidefix` (lib) due to 8 previous errors
  💥 maturin failed
    Caused by: Failed to build a native library through cargo
    Caused by: Cargo build finished with "exit status: 101": `"cargo" "rustc" "--release" "--features" "static" "--features" "python" "--features" "extension-module" "--features" "fast-index" "--target" "x86_64-unknown-linux-gnu" "--manifest-path" "/home/conda/staged-recipes/build_artifacts/hidefix_1692063824569/work/Cargo.toml" "--message-format" "json" "--lib" "--crate-type" "cdylib"`
  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:50:12
     |
  50 | #![feature(test)]
     |            ^^^^


  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:51:12
     |
  51 | #![feature(cow_is_borrowed, array_methods)]
     |            ^^^^^^^^^^^^^^^


  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:51:29
     |
  51 | #![feature(cow_is_borrowed, array_methods)]
     |                             ^^^^^^^^^^^^^


  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:52:12
     |
  52 | #![feature(assert_matches)]
     |            ^^^^^^^^^^^^^^


  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:53:12
     |
  53 | #![feature(slice_group_by)]
     |            ^^^^^^^^^^^^^^


  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:54:12
     |
  54 | #![feature(mutex_unlock)]
     |            ^^^^^^^^^^^^


  error[E0554]: `#![feature]` may not be used on the stable release channel
    --> src/lib.rs:55:12
     |
  55 | #![feature(new_uninit)]
     |            ^^^^^^^^^^


  error: aborting due to 7 previous errors


  For more information about this error, try `rustc --explain E0554`.

  Error: command ['maturin', 'pep517', 'build-wheel', '-i', '/home/conda/staged-recipes/build_artifacts/hidefix_1692063824569/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python', '--compatibility', 'off'] returned non-zero exit status 1
  error: subprocess-exited-with-error
  
  × Building wheel for hidefix (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /home/conda/staged-recipes/build_artifacts/hidefix_1692063824569/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python /home/conda/staged-recipes/build_artifacts/hidefix_1692063824569/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /tmp/tmpoer5g6nf
  cwd: /home/conda/staged-recipes/build_artifacts/hidefix_1692063824569/work
  Building wheel for hidefix (pyproject.toml): finished with status 'error'
  ERROR: Failed building wheel for hidefix
Failed to build hidefix
ERROR: Could not build wheels for hidefix, which is required to install pyproject.toml-based projects
```
So can confirm that `rust_dev` is required.

Reply:
Thanks for checking :+1: